bm-instance-agent/bm_instance_agent/common/utils.py

```
@@ -86,6 +86,8 @@ def query_link(iface):
         if_index = None
         if isinstance(iface, int):
             if_index = iface
+        # TODO: -> python3:
+        # elif isinstance(iface, str):
         elif isinstance(iface, (str, unicode)):
```

Ruff (0.15.4)

[error] Undefined name `unicode`

(F821)